src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -116,19 +113,10 @@
     <Compile Include="$(CommonPath)\Interop\Windows\Winsock\SafeFreeAddrInfo.cs">
       <Link>Interop\Windows\Winsock\SafeFreeAddrInfo.cs</Link>
     </Compile>
-  </ItemGroup>
-  <!-- Windows : Win32 only -->
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(TargetGroup)' != 'uap'">
     <Compile Include="$(CommonPath)\System\Net\ContextAwareResult.Windows.cs">
```

Comment:
Nit: I think this file would be better placed somewhere in line 52 to match the comments.